blog_system/urls.py

```diff
@@ -0,0 +1,9 @@
+from django.contrib import admin
+from django.urls import path, include
+from debug_toolbar.toolbar import debug_toolbar_urls
+
+urlpatterns = [
+    path("admin/", admin.site.urls),
+    path("blog/", include("blog.urls", namespace="blog")),
+    path("", include("django.contrib.auth.urls")),
+] + debug_toolbar_urls()
```

Comment:
The way you are adding debug_toolbar_urls to urlpatterns is incorrect. Instead of using `debug_toolbar_urls()`, use `path('__debug__/', include(debug_toolbar_urls))`. This will ensure the debug toolbar is correctly included in your urlpatterns.